BUILD.md

```diff
@@ -125,6 +125,7 @@ $ make -C site build
 
 This will auto-generate [Markdown](https://en.wikipedia.org/wiki/Markdown) files with documentation for currently supported Kopia CLI subcommands and store them under `site/content/docs/Reference/Command-Line` and then generate the website which is stored in `site/public`.
 
+
```

Comment:
nit; drop this

maybe your editor does this automatically ? Some of my vim plugins used to do this until I configured them to not do auto formatting on existing files.

Reply:
@Shrekster I added to trigger a test.  I'll remove it.